Appraisal: fix draggability of arranged files

replaceafill opened the pull request:
This PR fixes the logic to mark nodes in the Backlog pane of the Appraisal tab as not draggable.

Connected to https://github.com/archivematica/Issues/issues/821

ross-spencer commented on src/dashboard/tests/test_ingest.py:
```diff
@@ -76,3 +77,187 @@ def test_add_metadata_files_view(self):
         assert title in response.content
+
+
+def _assert_file_node_properties_match_record(file_node, record):
```
Nice idea to have an umbrella function for a group of validation tests.

ross-spencer commented on src/dashboard/tests/test_ingest.py:
```diff
@@ -76,3 +77,187 @@ def test_add_metadata_files_view(self):
+    assert file_node["relative_path"].decode("base64") == record["relative_path"]
+    # the node title is the encoded file name
+    assert (
+        file_node["title"].decode("base64") == record["relative_path"].rsplit("/", 1)[1]
```
Would `os.path.sep` be better to replace `/` or even, `os.path.basename()` satisfy what you're doing here?

replaceafill replied:
Ah thanks!

ross-spencer commented on src/dashboard/tests/test_ingest.py:
```diff
@@ -76,3 +77,187 @@ def test_add_metadata_files_view(self):
+    assert file_node["not_draggable"]
+
+
+def test_appraisal_tab_node_draggability_metadata():
```
Just a question about tests vs. visibility in the UI - the metadata dirs don't show up in the UI do they? (same for some of the "special" directories below?) 

(I can't see them, either way I do like this test being here for the future)

replaceafill replied:
You need to check the `Show metadata & logs directories?` checkbox before pressing the `Search transfer backlog` button in the Appraisal tab:

![Captura de pantalla de 2019-07-30 22-47-03](https://user-images.githubusercontent.com/560781/62184610-0ff9dc00-b31c-11e9-9c6c-6b7a7b2ae702.png)

ross-spencer replied:
Oh! Well, I've learned something new! (two things if you include I should click more buttons in the UI :) Thanks @replaceafill.

ross-spencer reviewed (approved):
Looks good to me @replaceafill and I tested this following the instructions in the issue and before the fix I can generate an error box at the top of the screen, but after the fix, I cannot even move the file. Nice work.

replaceafill committed:
Appraisal: fix draggability of arranged files

This fixes the logic to mark nodes in the Backlog pane of the
Appraisal tab as not draggable.
